doc/2/index.md

```diff
@@ -6,4 +6,4 @@ title: Kuzzle IoT - Device Manager
 description: Kuzzle IoT - Device Manager Documentation
 ---
 
-<Redirect to="guides/devices" />
\ No newline at end of file
+<Redirect to="controllers/asset" />
```

Comment:
When I explain KDM in speaking, I  start to explain "what is an asset" before "what is a device", because the goal of KDM is to store information about assets and device is just a tool to obtain information of the asset.
But in a written official documentation, perhaps it's better to begin with devices.